### ISSUE TITLE
There are some folders in .venv accidentally deleted even with param --exclude-envs

### ISSUE THREAD

**Issue by strollo** [bittner/pyclean#48]
Executing with `--exclude-envs` still there is some folder cleaned that breaks pip installation on .venv

In particular this folder is removed from .venv making pip no more working:

```
[INFO] remove directory [build - Python]: ./.venv/lib/python3.9/site-packages/pip/_internal/operations/build
```

The proper folder to remove should be instead `./.venv/lib/python3.9/site-packages/pip/_internal/operations/build/__pycache__`

**Comment by strollo** [comment]
sorry not an issue, my fault.

**Comment by bittner** [comment]
Thanks for reporting your issue, anyway! Glad you could resolve it all by yourself.

Can you describe what was the problem and how you fixed it? If it is a pitfall that we can make PyClean to safeguard against we may add some related code to the code base.

**Comment by strollo** [comment]
Sorry @bittner for the inconvenience.
Essentially I discovered a problem launching in nox the session for cleaning up caches.
I did not realize that I was using a plugin cleanpy in place of this pyclean.
So my first attempt was to take your code and trying to reproduce by extracting from your sources the logic. And it was no more cleaning the wrong folder.... after I while I realized I was using another plugin :)

Now with yours is perfectly working so no bug was present from the beginning.

Sorry again!
